dipy/segment/tests/test_feature.py

```diff
@@ -76,20 +81,24 @@ def extract(self, streamline):
                 # Test method extract
                 features = feature.extract(s)
                 assert_equal(features.shape, (nb_points, s.shape[1]))
-                assert_array_almost_equal(features, set_number_of_points(s, nb_points))
+                assert_array_almost_equal(
+                    features, set_number_of_points(
```

Comment:
I prefer this as two lines:

assert_array_almost_equal(features, 
set_number_of_points(s, nb_points)

with the second line properly indented.